script/build-artifact.sh

```diff
@@ -56,7 +62,10 @@ cp "${REPODIR}"/LICENSE "${EXTRAS_STAGING}"
 mkdir -p "${STAGING}"/bin "${EXTRAS_STAGING}"/bin
 cp "${BINDIR}"/spire-server "${STAGING}"/bin
 cp "${BINDIR}"/spire-agent "${STAGING}"/bin
-cp "${BINDIR}"/k8s-workload-registrar "${EXTRAS_STAGING}"/bin
+# Exclude registrar from windows artifact
+if [ $OS != "windows" ]; then 
+    cp "${BINDIR}"/k8s-workload-registrar "${EXTRAS_STAGING}"/bin
+fi
 cp "${BINDIR}"/oidc-discovery-provider "${EXTRAS_STAGING}"/bin
 
 # Create the tarballs and checksums
```

Comment:
I think zip is the preferred encoding for windows....

Reply:
I udpate it to use Compress-archive and store it as zip on windows